src/main/resources/lang/french.lang

```diff
@@ -0,0 +1,190 @@
+	check error: <red>Une erreur s'est produite lors de la vérification de la dernière version de Skript :<light red> %s
+	running latest version: Vous utilisez actuellement la dernière version stable de Skript.
+	running latest version (beta): Vous utilisez actuellement une version <i>bêta<r> de Skript et aucune nouvelle version <i>stable<r> n'est disponible. Notez que vous devez mettre à jour les nouvelles versions bêta manuellement !
+	update available: Une nouvelle version de Skript est disponible : <gold>%s<reset> (vous utilisez actuellement <gold>%s<reset>)
+	downloading: Téléchargement de Skript <gold>%s<reset>...
+	download in progress: La dernière version de Skript est en cours de téléchargement.
+	download error: <red>Une erreur s'est produite lors du téléchargement de la dernière version de Skript :<light red> %s
+	downloaded: La dernière version de Skript a été téléchargée ! Redémarrez le serveur ou utilisez /reload pour appliquer les changements.
+	internal error: Une erreur interne s'est produite lors de la vérification de la dernière version de Skript. Veuillez consulter le journal du serveur pour plus de détails.
+	custom version: Vous utilisez actuellement une version personnalisée de Skript. Aucune mise à jour ne sera installée automatiquement.
+	nightly build: Vous utilisez actuellement une version de développement de Skript. Aucune mise à jour ne sera installée automatiquement.
+
+# -- Commands --
+commands:
+	no permission message: Vous n'avez pas la permission d'utiliser cette commande
+	cooldown message: Vous utilisez cette commande trop souvent, veuillez réessayer plus tard
+	executable by players: Cette commande ne peut être utilisée que par des joueurs
+	executable by console: Cette commande ne peut être utilisée que par la console
+	correct usage: Utilisation correcte :
+	invalid argument: Argument invalide <gray>'%s<gray>'<reset>. Sont autorisés :
+	too many arguments: Cette commande ne peut accepter qu'un seul %2$s.
+	internal error: Une erreur interne s'est produite lors de la tentative d'exécution de cette commande.
+	no player starts with: Il n'y a aucun joueur en ligne dont le nom commence par '%s'
+	multiple players start with: Il y a plusieurs joueurs en ligne dont le nom commence par '%s'
+
+# -- Hooks --
+hooks:
+	hooked: Connecté avec succès à %s
+	error: Impossible de se connecter à %1$s. Cela peut se produire si Skript ne prend pas en charge la version installée de %1$s
+
+# -- Aliases --
+aliases:
+	# Errors and warnings
+	empty string: '' n'est pas un item type
+	invalid item type: '%s' n'est pas un item type
+	empty name: Un alias doit avoir un nom
+	brackets error: Utilisation incorrecte d'accolades
+	not enough brackets: La section commençant au caractère %s ('%s') doit être fermée
+	too many brackets: Le caractère %s ('%s') ferme une section inexistante
+	unknown variation: La variation %s n'a pas été défini avant
```

Comment:
I don't really know if this is better or clearer, but I give here another possibility: `La variante %s n'a pas été défini avant`